_vector-search/ai-search/building-agentic-search-flows.md

```diff
@@ -25,27 +28,99 @@ To configure new agents, be sure to first provision appropriate models. For work
 
 ### Ingest data
 
-Ensure that you have a sufficient number of documents in your cluster to reasonably evaluate your agentic searches. For more information, see [Agentic search]({{site.url}}{{site.baseurl}}/vector-search/ai-search/agentic-search/).
+Ensure that you have a sufficient number of documents in your cluster to reasonably evaluate your agentic searches.
 
-## Example: Product search with GPT-5
+## Accessing the plugin
+
+To access the plugin, go to **OpenSearch Dashboards** and select **OpenSearch Plugins** > **AI Search Flows** from the top menu.
+
+## Configuring agents
 
-This example uses a deployed OpenAI GPT-5 model described in [this documentation]({{site.url}}{{site.baseurl}}/vector-search/ai-search/agentic-search/agent-customization/#gpt-5-recommended).
+Agents are very customizable and can be configured in many different ways depending on your target use case. The following image shows a fully-configured conversational agent.
+
+![Agent configuration]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agent-configuration.png)
+
+### Agent types
+
+`Flow` agents are optimized for speed and simplicity in query generation. `Conversational` agents can be configured with more tools and functionality for deeper thinking. Agent types can only be configured **once** during agent creation.
+
+When configuring `Flow` agents, certain models may require manually adding an appropriate `response_filter` in the **Query Planning** tool. For more details, see [Register a flow agent]({{site.url}}{{site.baseurl}}/vector-search/ai-search/agentic-search/flow-agent/#step-4-register-a-flow-agent). Currently, only `OpenAI` and `Amazon Bedrock Converse` are supported. If you would like native support for other providers, please open a [GitHub issue](https://github.com/opensearch-project/dashboards-flow-framework/issues).
 {: .note}
 
-To build and test your agentic search workflow in OpenSearch Dashboards, follow these steps:
+### Models
+
+In `Conversational` agents, models are responsible for query generation and tool orchestration. In `Flow` agents, models are used strictly for query generation. Additionally, different models are typically optimized for different scenarios: cost-efficient, fast inference versus resource-intensive, deep-thinking approaches.
```

Comment:
models are responsible for intelligent thinking in terms of picking up the right set of tools, connecting to external sources, searching on web and then finally generating query based on the context gathered?

In flow agents, tools just run sequentially and can be best used to generate queries?

Reply:
Instead of 
```
and models are typically only used for query generation
```
model present in QPT is used for query generation?

Reply:
agreed. updated